driver/src/test/java/org/kaazing/k3po/driver/internal/RobotIT.java

```diff
@@ -468,6 +471,129 @@ public void shouldEcho() throws Exception {
     }
 
     @Test
```

Comment:
I think we were trying to implement these kind of tests using the `@TestSpecification` framework that mimics the Junit API.  I guess it's ok to do it this way, but was there a specific reason you chose this class functionally over the other.  If not, lets add deprecated to this class and explain the reason (Test framework is similar to junit and takes nearly identical code paths).  I believe K3poIT has examples...